test/bdd/bddtests_test.go

```diff
@@ -79,7 +81,29 @@ func runBDDTests(tags, format string) int { //nolint: gocognit
 						panic(fmt.Sprintf("Invalid value found in 'TEST_SLEEP': %s", e))
 					}
 				}
-				fmt.Printf("*** testSleep=%d", testSleep)
+
```

Comment:
I don't understand this logic

Reply:
I'll remove the debug printlns, is that it? Otherwise this is similar to the bdd test in trustbloc-did-method, where we start certain servers before the config files are generated, and start other servers after.